### PR TITLE
Add rake tasks to titleize client names; anonymize clients

### PR DESCRIPTION
**Pull request by gigxz**
and rubocop autocorrect the task file.

### CODE REVIEW

File: lib/tasks/grda_warehouse.rake [hunk 1]
```diff
@@ -218,13 +218,14 @@ namespace :grda_warehouse do
...
+  task :anonymize_client_names, [:path] => [:environment, 'log:info_to_stdout'] do |_task, args|
     raise 'path is required' unless args.path.present?
+
     path = args.path
     file = File.join(path, 'Client.csv')
     CSV.open("#{file.gsub('.csv', '.anon.csv')}", 'wb') do |csv|
       CSV.foreach(file, headers: true) do |row|
-        csv << row.headers() if $. == 2
+        csv << row.headers if $. == 2
         row['FirstName'] = "First_#{row['PersonalID']}"
```
Comment (gigxz) [inline lib/tasks/grda_warehouse.rake]: @eanders do you mind if I update this task to use the Faker code instead? Makes more sense to run it on the CSV before importing it, but I want the realistic data.

Comment (eanders) [reply lib/tasks/grda_warehouse.rake]: I think that's fine, we don't seem to use this anywhere currently.

Comment (eanders) [reply lib/tasks/grda_warehouse.rake]: It might make sense to cleanup the SSN in this as well.  Seems odd we'd change the names but not the SSNs.

**Review by eanders — APPROVED** [review]

File: lib/tasks/grda_warehouse.rake (hunk 1 above)
Comment (gigxz) [reply lib/tasks/grda_warehouse.rake]: I will move the ssn/dob cleanup code up here as well.

**Pull request merged**